packages/editor/CodeMirror/CodeMirrorControl.test.ts

```diff
@@ -57,4 +57,47 @@ describe('CodeMirrorControl', () => {
 		expect(control.execCommand('myTestCommand')).toBe('test');
 		expect(command).toHaveBeenCalledTimes(1);
 	});
+
+	it('should toggle comments', () => {
+		const control = createEditorControl('Hello\nWorld\n');
+		control.select(1, 5);
+
+		control.execCommand('toggleComment');
+		expect(control.getValue()).toBe('<!-- Hello -->\nWorld\n');
+
+		control.execCommand('toggleComment');
+		expect(control.getValue()).toBe('Hello\nWorld\n');
+	});
+
+	it('should delete line', () => {
+		const control = createEditorControl('Hello\nWorld\n');
+		control.setCursor(1, 0);
+
+		control.execCommand('deleteLine');
+		expect(control.getValue()).toBe('Hello\n');
+	});
+
+	it('should duplicate line', () => {
```

Comment:
While tests for comment and delete could make sense here, it might make sense to create `duplicateLine.test.ts` and `sortSelectedLines.test.ts` for organizational purposes.

However, it probably wouldn't make sense to construct a `CodeMirrorControl` in such files (I think a `duplicateLine.test.ts` test should pass independently of whether `duplicateLine` is registered as an editor command).

Reply:
should I keep these tests here then? or would checking the behaviour the .test.ts only be enough?

Reply:
I suggest moving tests for `duplicateLine` and `sortSelectedLines` to `duplicateLine.test.ts` and `sortSelectedLines.test.ts` files.